ush/predef_grid_params.yaml

```diff
@@ -61,7 +61,7 @@
   ESGgrid_NY: 131
   ESGgrid_PAZI: 0.0
   ESGgrid_WIDE_HALO_WIDTH: 6
-  DT_ATMOS: 40
+  DT_ATMOS: 180
   LAYOUT_X: 5
```

Comment:
I don't see specific modifications to use smaller time steps when running with RAP/HRRR physics-based SDFs.  If we are counting on users to manually modify the dt_atmos value in their config files when using those SDFs, some people won't understand the need to do that, and the default time step will be too large and fail.  This was the original reason we had time step defaults set to lower values.  I recall you mentioning that there is now a method through which we can automatically decrease the time step when certain SDFs are used?  Can this be implemented?